eng/pipelines/mgmt.yml

```
@@ -201,10 +210,13 @@ stages:
       - template: /eng/pipelines/templates/jobs/ci.mgmt.yml
         parameters:
           Scope: ${{ parameters.Scope }}
-          ShouldPublish: ${{ parameters.ShouldPublish }}
+          ShouldPublishToNuget: ${{ parameters.ShouldPublishToNuget }}
```

I don't think we need to distinguish between the type of publishing in this template. We could instead do something like:
```suggestion
  ${{ if or(eq(parameters.ShouldPublishToNuget, 'true'), eq(parameters.ShouldPublishToDevOps, 'true')) }}:
          ShouldPublish: true
```